CONTRIBUTING.md

```diff
@@ -55,6 +55,34 @@ Whenever running tests in torchtune, favor using the command line flags as much
 
 Note that the above flags can be combined with other pytest flags, so e.g. `pytest tests -m integration_test -k 'test_loss'` will run only recipe tests matching the substring `test_loss`.
 
+> [!NOTE]
+> For contributors who are developing on MPS, you may find that several of our tests fail due to precision differences falling outside of numerical tolerances on our tests. As of 27/09/2024 the following tests will fail for this reason:
+<details>
+  <summary>Failed Tests</summary>
```

Comment:
Commented in Discord, but why not just (a) add a new method in test_utils.py, (b) decorate each of these test cases with it, and (c) point to an issue tracking all of them in case someone wants to help debug any individual one?